spot_rl_experiments/experiments/skill_test/test_sem_place.py

```diff
@@ -4,56 +4,160 @@
 
 
 import copy
+import time
 
 import cv2
+import magnum as mn
 import numpy as np
 import quaternion
 import rospy
+import sophus as sp
 from bosdyn.client.frame_helpers import get_a_tform_b
-from perception_and_utils.utils.generic_utils import map_user_input_to_boolean
 from scipy.spatial.transform import Rotation
 from spot_rl.envs.skill_manager import SpotSkillManager
 from spot_rl.utils.construct_configs import construct_config
-from spot_wrapper.spot import SpotCamIds, image_response_to_cv2
+from spot_rl.utils.search_table_location import (
+    convert_point_in_body_to_place_waypoint,
+    detect_place_point_by_pcd_method,
+    plot_intel_point_in_gripper_image,
+)
+from spot_wrapper.spot import Spot, SpotCamIds, image_response_to_cv2
 
 if __name__ == "__main__":
+    from perception_and_utils.utils.generic_utils import map_user_input_to_boolean
 
     # Know which location we are doing experiments
-    in_fremont_lab = map_user_input_to_boolean("In Fremont location ? Y/N ")
-    if in_fremont_lab:
+    in_fre_lab = True  # map_user_input_to_boolean("Are you Tushar in FRE? Y/N ")
+    if in_fre_lab:
         # at FRE
         place_target = "place_taget_test_table"
+        place_target_before = "place_taget_test_table_before"
     else:
         # at NYC
-        place_target = "test_desk"
+        place_target = "high_shelf"  # "high_shelf", "test_desk" #"shelf" #"drawer_pretend_to_be_dish_washer"  # "test_semantic_place_table"
 
-    # Initialize the skill manager
-    spotskillmanager = SpotSkillManager(use_mobile_pick=True, use_semantic_place=True)
+    spotskillmanager = SpotSkillManager(use_mobile_pick=False, use_semantic_place=False)
+    # spotskillmanager.gaze_controller.reset_skill("bottle")
 
-    # Start testing
-    contnue = True
+    spot = spotskillmanager.spot
+
+    # spotskillmanager.pick("bottle")
+    contnue = True  # map_user_input_to_boolean("Should we continue ?")
+
+    config = construct_config()
     while contnue:
-        if in_fremont_lab:
+        if in_fre_lab:
+            # pass
```

Comment:
Hi, Tushar. I think if we want to use the function of estimation of place target, we would like to move the logics to inside skill_manager.py. So user does not need to worry about how to do this.